docs/web-servers/squid/squid-http-proxy-centos-8/index.md

```diff
@@ -0,0 +1,234 @@
+{{< /note >}}
+
+1.  Ensure that your system is up-to-date:
+
+        sudo yum update && sudo yum upgrade
+
+1.  Install Squid using the `yum` software package manager:
+
+        sudo yum install squid
+
+1.  Copy the original configuration file to keep as a backup:
+
+        sudo cp /etc/squid/squid.conf /etc/squid/squid.conf.default
+
+    {{< note >}}
+The Squid configuration file includes comprehensive documentation in its commented lines, along with several uncommented rules that will remain active. These default rules should not be modified while you are following this guide. To gain a deeper understanding of Squid's options and default settings, you can review the full configuration file.
+{{< /note >}}
+
+## Configure Client Access
+
+Now that you have Squid installed on your Linode, you can configure ways for it to accept connections and serve as an HTTP proxy. The following sections provide different ways for your Squid HTTP proxy to authenticate client connections. You can configure Squid to use either or both authentication methods.
+
+### IP Address Authentication
+
+A simple way to use Squid as an HTTP proxy is to use a client's IP address for authentication.
+
+1.  Edit the Squid configuration file and add the following lines at the beginning of the file:
+
+    {{< file "/etc/squid/squid.conf" >}}
+acl client src 192.0.2.0 # Home IP
+http_access allow client
+{{< /file >}}
+
+    Replace `client` with a name that identifies the client computer that will connect to your Squid HTTP proxy, then replace `192.0.2.0` with the client computer's IP address. You can also update the optional comment `# Home IP` to further describe the client.
+
+1.  Alternatively, you can configure multiple clients by adding new `acl` lines to `/etc/squid/squid.conf` and including them in the `http_access allow` line as follows:
+
+    {{< file "/etc/squid/squid.conf" >}}
+acl client1 src 192.0.2.0 # Home IP
+acl client2 src 192.0.2.1 # Work IP
+http_access allow client1 client2
+{{< /file >}}
+
+    Replace `client1` and `client2` with names that identify the client computers, then replace `192.0.2.0` and `192.0.2.1` with their corresponding IP addresses. Update the optional comments `# Home IP` and `# Work IP` with accurate descriptions to help keep track of multiple clients. Access to the proxy is granted by adding the names defined by each `acl` to the `http_access allow` line.
+
+### User/Password Authentication
+
+You can also configure your Squid HTTP proxy to accept authentication with usernames and passwords.
+
+1.  Install `htpasswd` by installing the Apache utility programs. If you have installed Apache on your Linode, you will already have it and can skip this step.
+
+        sudo yum install httpd-tools
+
+1.  Create a file to store Squid users and passwords:
+
+        sudo touch /etc/squid/squid_passwd
+
+1.  Change ownership of the password file:
```

Comment:
Is "squid" the name of the limited user account we set up with Securing Your Server or is this a Squid user? I ask because in the other guides it's "proxy," and I wasn't sure.

Reply:
Yes, just confirmed that on CentOS, the user is `squid` rather than `proxy`.

```
$ sudo chown proxy /etc/squid/squid_passwd
chown: invalid user: ‘proxy’
```